script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9.txt

```diff
@@ -69,7 +69,7 @@ If you want to use a phone, use the one provided in the lobby. If you want to do
 Ending the conversation, Akiha walks out of the dining room.
 <Page20>
 I'm standing there in shock for the umpteenth time today, and the biggest one yet.
-It's not that I'm a sore loser, but I'm not the type of person who uses a mobile phone that much.
+It's not that I'm a sore loser, and I'm not the type of person who uses a mobile phone that much either.//負け惜しみではないが、俺はそこまで携帯を愛用するタイプではない。
```

Comment:
The original is correct imo.
The が in ではないが signifies the but for me

Reply:
It could be a "but", but then it doesn't make sense for me. He says that he's not a sore loser, so he doesn't particularly mind his phone being taken, and then says that he doesn't use his phone a lot. There's no opposition between the two sentences.

Reply:
I kinda disagree. I interpreted this line as being like "I don't care about winning this argument but it's unreasonable since I'm not the type to use his phone much" since we know Shiki has some choice words for those types and he's being defensive of being lumped in with that crowd. Maybe that's overthinking it.

Reply:
My head is hurting because of this stupid sentence.